CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Added
 
+- Removed protractor related configuartion from ini.sh in Ionic quickstarter ([#885](https://github.com/opendevstack/ods-quickstarters/issues/885))
```

Comment:
I'd change the sentence to the following and I'd also recommend to move it down into the `### Modified` or `### Fixed` section.

```suggestion
- Removed protractor-related configuration from `ini.sh` in Ionic quickstarter ([#885](https://github.com/opendevstack/ods-quickstarters/issues/885))
```